doc/api/stream.md

```diff
@@ -1445,9 +1445,9 @@ added: v8.0.0
 -->
 
 * `callback` {Function} Call this function (optionally with an error
-  argument) when you are done writing any remaining data.
+  argument) when finished writing any remaining data.
 
-Note: `_final()` **must not** be called directly.  It MAY be implemented
+*Note*: `_final()` **must not** be called directly. It MAY be implemented
```

Comment:
Optional nit: We probably want to avoid ALL CAPS. Probably best to emulate the stayling of `must not` here for `may`. Then this line merely has two typographies for emphasis (italics and bold), and not three.